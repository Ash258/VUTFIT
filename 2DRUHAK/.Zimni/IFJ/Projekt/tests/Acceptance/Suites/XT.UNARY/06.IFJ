' Print statement

scope
	print 42; -78;   142.2541;
	print -14287877.254121211  ;
	print 14.25;
end scope
