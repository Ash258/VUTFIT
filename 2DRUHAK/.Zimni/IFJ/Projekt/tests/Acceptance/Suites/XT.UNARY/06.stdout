 42-78 142.254-1.42879e+07 14.25